packages/@lwc/integration-karma/scripts/shared/options.js

```diff
@@ -28,7 +28,10 @@ const DISABLE_STATIC_CONTENT_OPTIMIZATION = Boolean(
     process.env.DISABLE_STATIC_CONTENT_OPTIMIZATION
 );
 const NODE_ENV_FOR_TEST = process.env.NODE_ENV_FOR_TEST;
-const API_VERSION = process.env.API_VERSION && parseInt(process.env.API_VERSION, 10);
+const LATEST_API_VERSION = 60;
```

Comment:
Will this need to be updated every single release? If so we should add a comment or document it somewhere so it doesn't slip through the cracks.

Reply:
Or we should grab this from [@lwc/shared/api-version.ts's HIGHEST_API_VERSION](https://github.com/salesforce/lwc/blob/94432b279559412effb29e3c1b9d7fb1ead8742d/packages/%40lwc/shared/src/api-version.ts#L21) or somewhere else central to avoid duplicating the constant.

Reply:
I didn't find `HIGHEST_API_VERSION`, thanks for pointing that out. I will make the change.